pom.xml

```diff
@@ -27,42 +37,42 @@
         </dependency>
     </dependencies>
     <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
rollback pom changes except adding dependencies
everything should works fine I think